environments/production/terraform.yml

```diff
@@ -418,6 +416,8 @@ pgbouncer_nlbs:
   - name: pgshard_nlb-production
     targets:
       - pgbouncer9-production
+      - pgbouncer6-production
+      - pgbouncer7-production
```

Comment:
What if you take the approach of first combining all of them so all traffic goes through all five (5, 6, 7, 8, 9), and then removing pgbouncer nodes one at a time? They way you have it here, `pgbouncer7` and `pgbouncer6` will be getting more total traffic than `pgbouncer9` which creates a weird imbalance.

Reply:
yes, Danny. we can do it. I will add nodes today and based on utilization will plan to remove it.